### PR TITLE
Downcasing emails when getting too

### PR DESCRIPTION
**Pull request by jacoblb64**
Current state
- We are already previously downcasing on creating and updating, stopping uppercase pollution in the db

Changes
- Now always downcasing any email param if present to the contacts endpoint
- Added unit tests to check GET and PUT case neutrality

### CODE REVIEW

**Review by justintmckibben — APPROVED** [review]

File: app/controllers/contacts_controller.rb [hunk 1]
```diff
@@ -69,6 +69,7 @@ def update_params
 
   def set_contact_by_email
     params.require(:email)
+    params[:email] = params[:email].downcase
```
Comment (justintmckibben) [inline app/controllers/contacts_controller.rb:72]: Could you actually create a separate method called downcase_email that runs before every every method that uses email?

Comment (jacoblb64) [reply app/controllers/contacts_controller.rb:72]: Sure

Comment (jacoblb64) [reply app/controllers/contacts_controller.rb]: Updated and added unit tests 😉

**Comment by justintmckibben** [comment]
We should be sure to merge accounts with uppercase in them in prod

**Review by justintmckibben — APPROVED** [review]

**Comment by jacoblb64** [comment]
> We should be sure to merge accounts with uppercase in them in prod

There's 4 users this is still the case for, one of which is you. Let's strategize the best way to do that :)

**Pull request merged**